operator/src/network/cas.rs

```diff
@@ -44,6 +45,18 @@ pub struct CasConfig {
     pub ganache_resource_limits: ResourceLimitsConfig,
     pub postgres_resource_limits: ResourceLimitsConfig,
     pub localstack_resource_limits: ResourceLimitsConfig,
+    pub cas_api: CasApiConfig,
```

Comment:
maybe update the guide with an example of this? I think the section explaining casResourceLimits could maybe have it? I think it's like this?

```yaml
# network configuration
---
apiVersion: "keramik.3box.io/v1alpha1"
kind: Network
metadata:
  name: small
spec:
  replicas: 2
  cas:
    image: ceramicnetwork/ceramic-anchor-service:latest
    casResourceLimits:
      cpu: "250m"
      memory: "1Gi"
    casApi:
      env: ...
 ```